app/templates/_has_wants_selector.html

```diff
@@ -0,0 +1,182 @@
+            <thead>
+                <tr>
+                    <th>Image</th>
+                    <th>SKU</th>
+                    <th>Description</th>
+                    <th>Quantity Wanted</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr class="only-edit" id="add-ppe-row-{{ name }}">
+                    <td colspan="4">
+                        <div
+                            style="display: flex; width: 100%; align-items: center; justify-content: space-between; flex-direction: row;">
+                            <button class="btn btn-default"
+                                onclick="window.document.location.reload(); return false">Cancel</button>
+                            <button class="btn btn-success start-add-ppe" data-target="{{ name }}">
+                                <i class="glyphicon glyphicon-plus" aria-hidden="true"></i> Add PPE Type
+                            </button>
+                            <button class="btn btn-primary">Save</button>
+                        </div>
+                    </td>
+                </tr>
+                {% for ppe in ppe_types %}
+                {% if ppe[name]|length > 0 %}
+                <tr>
+                    {{ active_ppe_row(ppe) }}
+                </tr>
+                {% endif %}
+                {% endfor %}
+            </tbody>
+        </table>
+    </form>
+</div>
+{% endmacro %}
+
+{% block has_wants_selector %}
+<div class="modal fade" tabindex="-1" role="dialog" id="add-ppe-modal">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span
+                        aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title">Add a PPE Type</h4>
+            </div>
+            <div class="modal-body">
+                <input type="search" style="width: 100%" class="form-control" placeholder="Search for a PPE Type" id="add-ppe-type-search"/>
+                <br/>
+                <table class="table table-striped" id="add-ppe-types">
+                    <thead>
+                        <tr>
+                            <th>Image</th>
+                            <th>SKU</th>
+                            <th>Description</th>
+                            <th>Actions</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for ppe in ppe_types %}
+                        {% if ppe.wants|length > 0 or ppe.has|length > 0 %}
+                            {% set hidden = "hidden" %}
+                        {% else %}
+                            {% set hidden = "" %}
+                        {% endif %}
+                        <tr data-searchable="{{ ppe.desc }} {{ ppe.sku }}" {{ hidden }} id="add-ppe-type-result-{{ ppe.id }}">
```

Comment:
This is very hacky, but it works: The way we implement searching for new PPE types is by having an attribute (`data-searchable`) on the table row containing all searchable content for the row. When the user types something in the input box, some JS just loops through each row and hide the ones who's `data-searchable` attributes don't contain whatever the user typed.